src/app/services/baw-api/site/sites.service.ts

```diff
@@ -15,17 +14,23 @@ import {
   IdOr,
   IdParam,
   IdParamOptional,
+  New,
   option,
   StandardApi,
 } from "../api-common";
 import { Filters } from "../baw-api.service";
-import { filterMock, showMock } from "../mock/api-commonMock";
 import { Resolvers } from "../resolver-common";
 
+type Orphans = "orphans";
+const Orphans: Orphans = "orphans";
+function shallowOption(x?: New | Filter | Empty | Orphans) {
+  return x ? x : Empty;
+}
+
 const projectId: IdParam<Project> = id;
 const siteId: IdParamOptional<Site> = id;
 const endpoint = stringTemplate`/projects/${projectId}/sites/${siteId}${option}`;
-const endpointShallow = stringTemplate`/sites/${siteId}${option}`;
+const endpointShallow = stringTemplate`/sites/${siteId}${shallowOption}`;
```

Comment:
We should consider the orphan route a different route/endpoint.
It will only have the `filter` option (not `new`)